packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelect.tsx

```diff
@@ -30,8 +30,15 @@ export const SingleEntitySelect = ({
     refs: [containerRef],
     callback: (event) => {
       event.stopImmediatePropagation();
-
-      onCancel?.();
+      // Do not cancel if we're in an input
+      if (
+        !(
```

Comment:
Can we create a variable weAreInAnInput instead of a comment ? It's always more maintainable.

Reply:
Or even weAreNotInAnHTMLInput, anything that's easy to read.

Reply:
We strive for no comments (if not TODO comments) and prefer explicit naming, even if it's long.

Reply:
Makes sense, thanks for this info @lucasbordeau